### ISSUE TITLE
Navigation Drawer Shows Incorrect Selection on Nougat

### ISSUE THREAD

**Issue by theck13** [Automattic/simplenote-android#421]
### Expected
The navigation drawer shows a light blue background with blue icon and text for the selected item for all API levels.

### Observed
The navigation drawer can have different highlighted and selected items for Nougat.  See the screenshots below for example.

![drawer](https://cloud.githubusercontent.com/assets/3827611/22226627/70dbb71c-e184-11e6-8dd2-5bdd6e410cee.png)

In the All Notes (Trash) screenshot, the All Notes item is selected while the Trash item's icon and text remains highlighted.  In the Trash (All Notes) screenshot, the Trash item is selected while the All Notes item's icon and text remains highlighted.

### Reproduced
1. Open navigation drawer.
2. Tap ***All Notes*** item.
3. Tap any note from ***All Notes*** list.
4. Tap back arrow in action bar.
5. Open navigation drawer.
6. Notice ***All Notes*** is highlighted and selected.
7. Tap ***Trash*** item.
8. Open navigation drawer.
9. Notice ***All Notes*** is highlighted and ***Trash*** is selected.
10. Close navigation drawer.
11. Tap any note from ***Trash*** list.
12. Tap back arrow in action bar.
13. Open navigation drawer.
14. Notice ***Trash*** is highlighted and selected.
15. Tap ***All Notes*** item.
16. Open navigation drawer.
17. Notice ***All Notes*** is selected and ***Trash*** is highlighted.

#### Tested
Google Pixel on Android 7.1.1 with Simplenote 1.5.5

**Comment by GeekerAndyLove** [comment]
@theck13 Same problem on Android 7.1. I found that the color will change after step "**12. Tap back arrow in action bar.**" in your Reproduced. And the color will change immediately after **Trash** being selected in Marshmallow. I found this from [here](https://developer.android.com/about/versions/nougat/android-7.0-changes.html)
> If an app posts Runnable tasks to a View, and the View is not attached to a window, the system queues the Runnable task with the View; the Runnable task does not execute until the View is attached to a window. This behavior fixes the following bugs:
If an app posted to a View from a thread other than the intended window’s UI thread, the Runnable may run on the wrong thread as a result.
If the Runnable task was posted from a thread other than a looper thread, the app could expose the Runnable task. 

I wonder if the change causes this bug?
